odoo/addons/test_inherits/models.py

```diff
@@ -72,5 +72,6 @@ class AnotherBox(models.Model):
 
     @api.constrains('val1', 'val2')
     def _check(self):
-        if self.val1 != self.val2:
-            raise ValidationError("The two values must be equals")
+        for box in self:
```

Comment:
This one can use any too :-)

Reply:
I've read that better to use this :

```suggestion
        if any(True for box in self if box.val1 != box.val2):
```